Components/Modal/modal.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <link rel="stylesheet" href="/css/nav.css">
+    <link rel="stylesheet" href="/css/doc.css">
+    <link rel="stylesheet" href="/Components/Modal/modal.css">
+    <link rel="stylesheet" href="/Components/Button/button.css">
+    <link rel="stylesheet" href="/prism.css">
+    <link rel="shortcut icon" href="../../images/Ozone-modified.png" type="image/x-icon">
+    <title>Ozone | Modal</title>
+</head>
+
+<body>
+    <div class="container">
+        <nav class="header">
+            <div class="left-nav">
+                <a href="../../index.html">
+                    <img src="../../images/Ozone.png" class="nav-img">
+                </a>
+            </div>
+            <ul class="right-nav">
+                <li>
+                    <a href="https://github.com/Megha-Joshi"><i class="fab fa-github"></i></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/megha-joshi24/"><i class="fab fa-linkedin"></i></a>
+                </li>
+                <li>
+                    <a href="https://twitter.com/megha24joshi"><i class="fab fa-twitter"></i></a>
+                </li>
+            </ul>
+        </nav>
+    </div>
+
+
+    <!-- Docs -->
+    <div class="main">
```

Comment:
Appreciation: it good to see that you are giving comment message , it helps to reviewer while reviewing code